doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -31,6 +31,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "# Contents\n",
+    "In this this tutorial, you will learn about:\n",
+    " 1. Reading parquet data,\n",
+    " 2. Creating Ray tasks to preprocess, train and evaluate data batches,\n",
+    " 3. Dividing data into batches and spawning a Ray task for each batch to be ran in parallel,\n",
+    " 4. The runtime-memory tradeoff between delegating data loading to each task as opposed to centralized data loading.\n",
```

Comment:
Can we add something like these as headers within the Walkthrough so that it's easier for the reader to understand what each step is in the walkthrough?